tests/tests/map_editor.spec.ts

```diff
@@ -128,7 +128,9 @@ test.describe("Map editor @oidc", () => {
         await AreaEditor.setListenerZoneProperty(page, `${browser.browserType().name()}SpeakerZone`.toLowerCase());
         await Menu.closeMapEditor(page);
         await Map.teleportToPosition(page, 4 * 32, 2 * 32);
-        await expect(page.locator('#cameras-container').getByText('Admin1')).toBeVisible();
+
+        await expect(page.getByText('You Reduce Toggle fullscreen')).toBeVisible();
```

Comment:
This test is not stable. 'You Reduce Toggle fullscreen' is some aggregated text based on the content of several div.
It's going to break very quickly. We need something better.